handlers/mainHandler.py

```diff
@@ -214,20 +225,18 @@ def asyncPost(self) -> None:
                     "Server has restarted.",
                 ) + serverPackets.banchoRestart(0)
             finally:
-                userToken = osuToken.get_token(requestTokenString)
```

Comment:
i have no idea why this was being fetch again here?

Reply:
hmm actually perhaps this is because `bancho_packets[packetID](userToken, packetData)` can modify it

Reply:
is it in-case one of the packet handlers modifies the session?

Reply:
re-added

Reply:
thinking for example L228, a packet may set kicked but we don't have the updated session..?